PYME/LMVis/Extras/pointwiseColoc.py

```diff
@@ -21,8 +21,18 @@ def OnPointwiseColoc(self, event):
         #from PYME import mProfile
         #mProfile.profileOn(['distColoc.py'])
         from PYME.Analysis.Colocalisation import distColoc
+        from PYME.DSView.modules.coloc import ColocSettingsDialog
+
+        chans = self.visFr.pipeline.colourFilter.getColourChans()[::1]
```

Comment:
What is the `[::1]` doing?

Reply:
Not sure. It's a historic thing that I just copied over.